src/bouwdossiers/tests/test_import.py

```diff
@@ -58,52 +58,40 @@ def test_prewabo_import(self):
         # 0363200000470955 Feike de Boerlaan 290 nummeraanduiding
         self.assertFalse('0363200000470955' in fdb.nummeraanduidingen)
 
-    def test_wabo_import(self):
-        batch.import_wabo_dossiers()
-        bd = models.BouwDossier.objects.get(dossiernr=51322878)
-        self.assertEqual(bd.stadsdeel, 'SDW')
-        self.assertEqual(bd.titel, "MK Sloopmelding asbestverwijdering bij mutatie- en klachtenonderhoud op het adres Van Hallstraat\n\t\t\t64 AMSTERDAM - (Portiekflat)")
-        self.assertEqual(bd.datering.strftime("%Y"), "2013")
-        self.assertEqual(bd.dossier_type, "sloopmelding")
-        self.assertEqual(bd.access, "RESTRICTED")
-        self.assertEqual(bd.source, "WABO")
-        self.assertEqual(bd.olo_liaan_nummer, 831835)
-        self.assertEqual(bd.wabo_bron, "SquitXO")
-        adres0 = bd.adressen.first()
-        self.assertEqual(adres0.straat, "Van Hallstraat")
-        self.assertEqual(adres0.huisnummer_van, 64)
-        self.assertEqual(adres0.huisnummer_tot, None)
-        self.assertEqual(adres0.openbareruimte_id, "0363300000005205")
-        self.assertEqual(adres0.verblijfsobjecten, ['0363010000839320'])
-        self.assertEqual(adres0.panden, ['0363100012133317'])
-        documenten = bd.documenten.order_by('id').all()
-        self.assertEqual(len(documenten), 13)
-        document12 = documenten[12]
-        self.assertEqual(document12.document_omschrijving, "Procesoverzicht")
-        self.assertEqual(document12.bestanden, ['SDW/ACTIVITY_DOCS/Procesinformatie sdw_51322878.pdf'])
-        self.assertEqual(document12.oorspronkelijk_pad, ['F:/INZAGEDOCS/SDW/ACTIVITY_DOCS/Procesinformatie sdw_51322878.pdf'])
+    # def test_wabo_import(self):
+    #     batch.import_wabo_dossiers()
+
+    #     # And another test for the WABO_centrum file from the new WABO server. The old wabo server is phased out.
```

Comment:
Liever de code gewoon verwijderen. Je kunt het altijd terug vinden in de git history

Reply:
Klopt. My bad.